packages/compass-workspaces/src/components/workspaces.tsx

```diff
@@ -181,15 +185,17 @@ const CompassWorkspaces: React.FunctionComponent<CompassWorkspacesProps> = ({
             iconGlyph: 'Shell',
             tabTheme: getThemeOf(tab.connectionId),
           } as const;
-        case 'Databases':
+        case 'Databases': {
+          const connectionName = getConnectionTitleById(tab.connectionId);
           return {
             id: tab.id,
-            connectionName: getConnectionTitleById(tab.connectionId),
+            connectionName,
             type: tab.type,
-            title: tab.type,
+            title: multiConnectionsEnabled ? connectionName : tab.type,
```

Comment:
We decided already that it's okay to have the same breadcrumb for this tab without the feature flag check, why are we introducing a difference here? Is it a requirement?

Reply:
when you say breadcrumb I think of subtitle, not the title. I don't recall us deciding on changing the title for all, but maybe I missed that for the same reason 😅 let me check on the channel